### PR TITLE
:sparkles: add description and finishedMessage for UserTask

### PR DESCRIPTION
**Pull request by roblen45**
**Changes:**

1. :sparkles: Add description and finishedMessage for UserTasks

PR: #229 

---

See the [examples](#example) for inspiration.

## How can others test the changes?

> Describe how others can test your changes (you can remove this section for typo fixes etc.)

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.).
- [x] I've rebased the `develop` branch with my branch before finishing this PR.
- [x] I've **summarized all changes** in a list above.
- [x] I've mentioned all **PRs, which relate to this one**.
- [x] I've prefixed my Pull Request title is according to [gitmoji guide](https://gitmoji.carloscuesta.me/).

## Example

<details>
<summary>
:warning: Before merging please click here to expand and follow the guide.
</summary>
<br>

Please use `:twisted_rightwards_arrows:` at the beginning of your merge commit title.

Example 1:

<pre>
<code>
:twisted_rightwards_arrows: :bug: Fix Wrong Text Decoration at ...
</code>
</pre>

To get your commit message, just copy the first part of this pull request.

Example 2:
<pre>
<code>
**Changes:**

- Fixes Wrong Text Decoration at ...
- Fixes some typos
- ...

**Issues:**

Closes #NumberOfFixedIssue

PR: #NumberOfThisPR
</code>
</pre>
</details>

## Emojis

<details>
<summary>
Expand for a list of most used Emojis.
</summary>
<br>

Please prefix your commit messages with an Emoji.

Ref: https://gitmoji.carloscuesta.me/

| Description              | Glyphe               | Emoji  |
|--------------------------|----------------------|--------|
| Bugfix                   | `:bug:`              | 🐛     |
| Fixing Security Issues   | `:lock:`             | 🔒     |
| Configuration releated   | `:wrench:`           | 🔧     |
| Cosmetic                 | `:lipstick:`         | 💄     |
| Dependencies Downgrade   | `:arrow_down:`       | ⬇️     |
| Dependencies Upgrade     | `:arrow_up:`         | ⬆️     |
| Formatting               | `:art:`              | 🎨     |
| Improving Performance    | `:zap:`              | ⚡️      |
| Initial commit           | `:tada:`             | 🎉     |
| Linter                   | `:rotating_light:`   | 🚨     |
| Miscellaneous            | `:package:`          | 📦     |
| New Feature              | `:sparkles:`         | ✨     |
| Refactoring Code         | `:recycle:`          | ♻️      |
| Releasing / Version tags | `:bookmark:`         | 🔖     |
| Removing Stuff           | `:fire:`             | 🔥     |
| Tests                    | `:white_check_mark:` | ✅     |
| Work In Progress (WIP)   | `:construction:`     | 🚧     |

</details>

### CODE REVIEW

**Comment by Vyperus** [comment]
Please choose a more active language in the PR description.

```
1. Add description and finishedMessage
```

**Comment by ElRaptorus** [comment]
Please cleanup the PRs body in the same manner as [this](https://github.com/process-engine/process_engine_contracts/pull/96) one.

File: src/model/parser/activity_parser.ts [hunk 1]
```diff
@@ -377,8 +391,8 @@ function getMethodInvocation(extensionProperties: Array<Model.Base.CamundaExtens
 }
 
 function findExtensionPropertyByName(propertyName: string,
```
Comment (ElRaptorus) [inline src/model/parser/activity_parser.ts]: Please move that parameter into a new line as well, or we'll get linter errors here.

File: src/model/parser/activity_parser.ts [hunk 2]
```diff
@@ -419,9 +433,9 @@ function parseCallActivities(processData: any): Array<Model.Activities.CallActiv
 }
 
 function parseSubProcesses(processData: any,
```
Comment (ElRaptorus) [inline src/model/parser/activity_parser.ts]: See above.

File: src/model/parser/activity_parser.ts [hunk 3]
```diff
@@ -269,6 +271,18 @@ function parseServiceTasks(processData: any): Array<Model.Activities.ServiceTask
 }
 
 function getPreferredControlForUserTask(userTaskRaw: Model.Activities.UserTask): string {
+  return getValueFromExtensionProperty('preferredControl', userTaskRaw)
```
Comment (ElRaptorus) [inline src/model/parser/activity_parser.ts]: Missing `;`.

File: src/model/parser/activity_parser.ts [hunk 4]
```diff
@@ -269,6 +271,18 @@ function parseServiceTasks(processData: any): Array<Model.Activities.ServiceTask
...
+}
+
+function getDescriptionForUserTask(userTaskRaw: Model.Activities.UserTask): string {
+  return getValueFromExtensionProperty('description', userTaskRaw)
```
Comment (ElRaptorus) [inline src/model/parser/activity_parser.ts]: Missing `;`.

File: src/model/parser/activity_parser.ts [hunk 5]
```diff
@@ -269,6 +271,18 @@ function parseServiceTasks(processData: any): Array<Model.Activities.ServiceTask
...
+}
+
+function getFinishedMessageForUserTask(userTaskRaw: Model.Activities.UserTask): string {
+  return getValueFromExtensionProperty('finishedMessage', userTaskRaw)
```
Comment (ElRaptorus) [inline src/model/parser/activity_parser.ts]: Missing `;`.

File: src/model/parser/activity_parser.ts [hunk 6]
```diff
@@ -269,6 +271,18 @@ function parseServiceTasks(processData: any): Array<Model.Activities.ServiceTask
...
+}
+
+function getValueFromExtensionProperty(name: string, userTaskRaw: Model.Activities.UserTask) {
```
Comment (ElRaptorus) [inline src/model/parser/activity_parser.ts]: Missing typedef for the function.

File: src/model/parser/activity_parser.ts [hunk 7]
```diff
@@ -402,7 +416,7 @@ function parseCallActivities(processData: any): Array<Model.Activities.CallActiv
       // NOTE: There is also a CMMN type, which is not supported yet.
       callActivity.type = Model.Activities.CallActivityType.BPMN;
       callActivity.calledReference = callActivityRaw.calledElement;
-      callActivity.bindingType = <Model.Activities.CallActivityBindingType> callActivityRaw[BpmnTags.CamundaProperty.CalledElementBinding];
+      callActivity.bindingType = <Model.Activities.CallActivityBindingType>callActivityRaw[BpmnTags.CamundaProperty.CalledElementBinding];
```
Comment (ElRaptorus) [inline src/model/parser/activity_parser.ts]: This change produces a linter error.

File: src/model/parser/activity_parser.ts [hunk 8]
```diff
@@ -477,13 +491,13 @@ function createActivityInstance<TActivity extends Model.Activities.Activity>(dat
...
+  eventDefinitionId: string,
+  eventDefinitions: Array<Model.EventDefinitions.EventDefinition>): TEventDefinition {
 
   const matchingEventDefintion: Model.EventDefinitions.EventDefinition =
     eventDefinitions.find((entry: Model.EventDefinitions.EventDefinition): boolean => {
       return entry.id === eventDefinitionId;
     });
 
-  return <TEventDefinition> matchingEventDefintion;
+  return <TEventDefinition>matchingEventDefintion;
```
Comment (ElRaptorus) [inline src/model/parser/activity_parser.ts]: This change produces a linter error.

**Pull request merged**